### ISSUE TITLE
Heavy debug tests?

### ISSUE THREAD

**Issue by WilkAndy** [idaholab/moose#11460]
## Rationale

Recently we noticed that a PorousFlow heavy test was failing in debug mode (Issue #11335).  This was an actual bug.  Can we have an additional test recipe: heavy test in debug mode?

## Description

We'll have to be careful with timeouts.

## Impact

Ability to check heavy tests in debug mode.

**Comment by brianmoose** [comment]
@permcody Do you want me to add heavy module testing in debug mode to the devel branch?

**Comment by permcody** [comment]
Not yet... I suspect we'll have a lot of timeouts. Maybe once we get this system more stable we can try it.

**Comment by brianmoose** [comment]
Yeah, I ran it on my mac, here are the failures:
```
FAIL combined/test:contact_verification/hertz_cyl/half_symm_q4.frictionless_aug FAILED (TIMEOUT)
FAIL combined/test:contact_verification/hertz_cyl/quart_symm_q4.frictionless_aug FAILED (TIMEOUT) [FINISHED]
FAIL combined/test:contact_verification/hertz_cyl/quart_symm_q8.frictionless_aug FAILED (TIMEOUT) [FINISHED]
FAIL porous_flow/examples/flow_through_fractured_media.coarse_3D FAILED (CRASH)
FAIL combined/test:poro_mechanics.borehole_heavy FAILED (TIMEOUT) [FINISHED]
FAIL combined/test:contact_verification/hertz_cyl/half_symm_q8.frictionless_aug FAILED (TIMEOUT)
FAIL porous_flow/examples/coal_mining.coarse FAILED (TIMEOUT) [FINISHED]
FAIL porous_flow/test:fluidstate.theis_brineco2 FAILED (TIMEOUT) [FINISHED]
FAIL combined/test:contact_verification/hertz_cyl/half_symm_q8.mu_0_2_aug FAILED (TIMEOUT) [FINISHED]
FAIL porous_flow/test:broadbridge_white.wli01 FAILED (TIMEOUT)
FAIL xfem/test:crack_tip_enrichment.crack_tip_enrichment_penny_crack_3d FAILED (TIMEOUT) [FINISHED]
DIFF combined/test:sliding_block/sliding/constraint.frictionless_aug FAILED (EXODIFF) [FINISHED,min_cpus=4]
```

**Comment by WilkAndy** [comment]
At least the only one that definitely crashed was the one i just fixed (Issue #11335).

**Comment by WilkAndy** [comment]
Closing due to lack of interest.  I run the dbg versions on my mac manually every once-in-a-while.
